packages/data-api/scripts/dropAnalyticsMaterializedView.sql

```diff
@@ -0,0 +1,39 @@
+do $$
+declare
+  tStartTime TIMESTAMP;
+  source_table TEXT;
+  source_tables_array TEXT[] := array['answer', 'survey_response', 'entity', 'survey', 'question', 'data_source'];
```

Comment:
Not asking for this to happen right now, but it would be good for us in the future to consolidate some of this logic with the createAnalyticsMaterializedView script to keep things DRY and avoid possible bugs 👍

Reply:
Yep that's true. Not going to address right now